Test/README.md

```diff
@@ -10,15 +10,15 @@ This document describes how to set up everything needed for running Windows sani
 
 ### CodiLime lab
 
-To deploy in local CodiLime VMWare lab, follow [this](https://juniper.github.io/contrail-windows-docs/contrail_deployment/Devenv_in_local_env/) document.
+To deploy in local CodiLime VMWare lab, follow [this](https://juniper.github.io/contrail-windows-docs/For%20developers/Developer%20guide/Setting%20up/Devenv_in_local_env/) document.
```

Comment:
spaces won't work?

Reply:
![image](https://user-images.githubusercontent.com/9220158/46863747-773fb180-ce18-11e8-9e9b-c5147790b010.png)
added a space between `Setting` and `up`, here's the result^

Reply:
quotes don't work either